packages/web/src/brandkit/common/Page.tsx

```diff
@@ -0,0 +1,240 @@
+import { SingletonRouter, withRouter } from 'next/router'
+import * as React from 'react'
+import { findNodeHandle, StyleSheet, View } from 'react-native'
+import MobileMenu from 'src/brandkit/common/MobileMenu'
+import Sidebar from 'src/brandkit/common/Sidebar'
+import Topbar from 'src/brandkit/common/TopBar'
+import { Cell, GridRow, Spans } from 'src/layout/GridRow'
+import { ScreenProps, ScreenSizes, withScreenSize } from 'src/layout/ScreenSize'
+import Footer from 'src/shared/Footer.3'
+import menu, { hashNav } from 'src/shared/menu-items'
+import { HEADER_HEIGHT } from 'src/shared/Styles'
+import { colors, standardStyles } from 'src/styles'
+
+const ROOT = menu.BRAND.link
+
+const LOGO_PATH = `${ROOT}/logo`
+
+const COLOR_PATH = `${ROOT}/color`
+
+const TYPE_PATH = `${ROOT}/typography`
+
+// const IMAGERY_PATH = `${ROOT}/key-imagery`
+const ICONS_PATH = `${ROOT}/icons`
+
+const PAGES = [
+  {
+    title: 'Introduction',
+    href: ROOT,
+    sections: [],
+  },
+  {
+    title: 'Logo',
+    href: LOGO_PATH,
+
+    sections: [
+      { title: 'Overview', href: `${LOGO_PATH}#${hashNav.brandLogo.overview}` },
+      { title: 'Space and Sizing', href: `${LOGO_PATH}#${hashNav.brandLogo.space}` },
+      { title: 'Backgrounds', href: `${LOGO_PATH}#${hashNav.brandLogo.backgrounds}` },
+    ],
+  },
+  {
+    title: 'Color',
+    href: COLOR_PATH,
+
+    sections: [
+      { title: 'Overview', href: `${COLOR_PATH}#${hashNav.brandColor.overview}` },
+      { title: 'Background Colors', href: `${COLOR_PATH}#${hashNav.brandColor.backgrounds}` },
+    ],
+  },
+  {
+    title: 'Typography',
+    href: TYPE_PATH,
+
+    sections: [
+      { title: 'Overview', href: `${TYPE_PATH}#${hashNav.brandTypography.overview}` },
+      { title: 'Type Scale', href: `${TYPE_PATH}#${hashNav.brandTypography.scale}` },
+    ],
+  },
+  {
+    title: 'Icons',
+    href: ICONS_PATH,
+    sections: [],
+  },
+  // {
```

Comment:
imagery will return in v1.1

Reply:
to reduce comments, maybe just define this object outside the array and then don't use it? (might have to ts-ignore)

Reply:
if i was gonna do that id just delete it, plan is to have version with imagery page in up by end of the week.